internal/util/grpcclient/client.go

```diff
@@ -328,12 +341,16 @@ func (c *ClientBase[T]) call(ctx context.Context, caller func(client T) (any, er
 		}
 		ret, callErr = caller(client)
 		if callErr != nil {
-			if funcutil.IsGrpcErr(callErr) ||
+			if funcutil.IsGrpcErr(callErr, codes.Unavailable, codes.Aborted, codes.FailedPrecondition) ||
 				IsCrossClusterRoutingErr(callErr) || IsServerIDMismatchErr(callErr) {
```

Comment:
are we pretty sure IsCrossClusterRoutingErr and IsServerIDMismatchErr throws with code 
codes.Unavailable, codes.Aborted, codes.FailedPrecondition?

Reply:
This condition lists all the case we might wanna reset the connection here